include/ppx/graphics_util.h

```diff
@@ -149,6 +149,14 @@ class TextureOptions
         const std::filesystem::path& path,
         grfx::Texture**              ppTexture,
         const TextureOptions&        options);
+
+    friend Result LoadFramesFromRawVideo(
```

Comment:
This method isn't used in this PR, but will be used later during the bandwidth tests (when productionized).